app/components/submission/ManuscriptMetadata/SubjectAreaDropdown.test.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import theme from '@pubsweet/elife-theme'
+
+import SubjectAreaDropdown from './SubjectAreaDropdown'
+
+const onChange = jest.fn()
+
+const MyDropdown = (
+  <SubjectAreaDropdown
+    label="My label"
+    name="My name"
+    onBlur={jest.fn()}
+    onChange={onChange}
+    savedValues={[]}
+    theme={theme}
+  />
+)
+
+describe('SubjectAreaDropdown component', () => {
+  it('renders label & select', () => {
+    const wrapper = shallow(MyDropdown).dive() // dive is necessary because SubjectAreaDropdown is wrapped by withTheme
```

Comment:
Long line comments anyone? Not a thing in javascript? :)

Reply:
"Not a thing"?

Are you just requesting that I put the comment on the line above? Or make the comment shorter? I'm not sure I understand what changes you're requesting?

Reply:
So in most programming languages there is a rule to limit the number of characters per line to 80. If you want to read more about it
https://en.wikipedia.org/wiki/Characters_per_line
https://softwareengineering.stackexchange.com/questions/148677/why-is-80-characters-the-standard-limit-for-code-width

I think eslint/prettier should try to split lines longer than 80 characters, not sure what happened here.

Limiting the line to 80 characters seems like a burden when using HTML for example, which is hard to split on multiple lines. That's why webdev seems more relaxed about this rule. However, personally I prefer lines shorter than 80 characters, it's a pain to watch the line wrap itself on multiple "lines" in my editor.

I would personally move the comment up a line, then if it's still longer than 80 characters I would split it into multiple lines like so
```
/**
* line 1
* line 2
* ...
*/
```

Reply:
I see. So you're requesting that I move the comment to the line above? No problem.